flink-connectors/flink-connector-aws-kinesis-data-streams/src/test/java/org/apache/flink/streaming/connectors/kinesis/util/AwsV2UtilTest.java

```diff
@@ -258,7 +263,12 @@ public void testCreateNettyHttpClientWithDefaults() {
 
         AwsV2Util.createHttpClient(clientConfiguration, builder, new Properties());
 
-        verify(builder).build();
+        AttributeMap defaultCustomAttribute =
```

Comment:
Why do we need this change?

Reply:
I changed that method to `buildWithDefault` with a custom attribute rather than `build`. This change relates to allowing users to set cert checking. This attribute instead of being set in the builder, is set when `buildWithDefault` is called, with an `AttributeMap` as parameter.